client/my-sites/site-monitoring/metrics-tab.tsx

```diff
@@ -152,19 +152,28 @@ export const MetricsTab = () => {
 			<TimeDateChartControls onTimeRangeChange={ handleTimeRangeChange }></TimeDateChartControls>
 			<SiteMonitoringLineChart
 				title={ __( 'Requests per minute & average response time' ) }
+				tooltip={ __(
+					'Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum.'
```

Comment:
**Requests per minute**: a line representing the number of requests received every minute.

 **Average Response Time**: a line that indicates the average time taken to respond to a request within that minute.